setup.py

```diff
@@ -39,6 +39,7 @@ def get_install_requires() -> List[str]:
         "scipy!=1.4.0" if sys.version[:3] == "3.6" else "scipy>=1.7.0",
         "sqlalchemy>=1.1.0",
         "tqdm",
+        "typing_extensions",
```

Comment:
```suggestion
        "typing_extensions>=3.10.0.0",
```

Without the version constraint, `pip install .[test]` or `pip install .[integration]` seemingly install old `typing_extensions`.

Reply:
Thank you for investigation! Let's try this change.